src/mono/msvc/libeventpipe.targets

```diff
@@ -7,142 +7,113 @@
   <ItemGroup Label="libeventpipe">
     <ClCompile Include="$(MonoSourceLocation)\mono\eventpipe\ep.c">
       <ExcludedFromBuild>$(ExcludeEventPipeFromBuild)</ExcludedFromBuild>
-      <CompileAs>CompileAsC</CompileAs>
```

Comment:
This change (and others in the .targets file) are just temporary for validation, correct?  Or do you intend to build EP as a C++for Mono?

Reply:
Not doing explicit specification mean that it will follow the global settings, so it will build using C if rest of runtime is build using C and C++ if rest is build using C++. By default we only build C based runtime in dotnet/runtime repro at the moment, but library can build using either C or C++ (fixed in this PR), so it could be embedded and used in C++ codebases as well.